subsys/net/lib/lwm2m/lwm2m_rd_client.c

```diff
@@ -464,7 +466,7 @@ int engine_trigger_bootstrap(void)
 	rd_client_message_free();
 	client.use_bootstrap = true;
 	client.trigger_update = false;
-	set_sm_state(ENGINE_INIT);
+	set_sm_state_delayed(ENGINE_INIT, 1000);
```

Comment:
Magic numbers should be avoided, we could have here for example `MSEC_PER_SEC` instead

Reply:
Thanks. Fixed.
I converted all `1000` values on calculations to `MSEC_PER_SEC`.
Then I defined two more macros, `DELAY_BEFORE_CLOSING` and `DELAY_FOR_ACK`